libcxx/include/__algorithm/adjacent_find.h

```diff
@@ -27,7 +32,7 @@ _LIBCPP_BEGIN_NAMESPACE_STD
 
 template <class _Iter, class _Sent, class _BinaryPredicate>
```

Comment:
We should call this `adjacent_find` from `ranges::adjacent_find` too.

This can be tackled as part of https://github.com/llvm/llvm-project/issues/105687 instead.

Reply:
I think it would be nice if this were addressed before we merge this patch. That way, this patch can actually add a short benchmark for `ranges::adjacent_find` too and we can confirm that we're improving both.